src/rating.css

```diff
@@ -0,0 +1,32 @@
+.product-card__rating {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 16px;
+}
+
+.product-card__rating-stars {
+  display: flex;
+}
+
+.product-card__rating-stars-4 {
+  background-image: url(images/star-active.svg);
+}
+
+.product-card__rating-stars-1 {
+  background-image: url(images/star.svg);
+}
+
+.product-card__rating-stars-4,
+.product-card__rating-stars-1 {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+}
+
+.product-card__rating-reviews {
+  font-style: bold;
+  font-weight: 400px;
+  font-size: 10px;
+  line-height: 14px;
+}
```

Comment:
delete this file as you have same file in blocks folder
and attach this file to your style.css